_patterns/bjorklund.md

```diff
@@ -1,57 +1,72 @@
 ---
-category: bjorklund 
+category: bjorklund
 layout: default
 ---
 
-If you give two numbers in parenthesis after an element in a pattern, then Tidal will
-distribute the first number of sounds equally across the second number of steps:
+If you give two numbers in parenthesis after an element in a pattern, then Tidal
```

Comment:
I reckon parenthesis is right, although to be ultra-pedantic it should probably just be "in brackets" as the concept of parenthesis doesn't really fit here..